control-board/src/bin/control/control.rs

```diff
@@ -492,7 +483,7 @@ impl<'a> Control<'a> {
                 self.ticks_since_packet = 0;
             }
         }
-        let wheel_vels = self.robot_model.robot_vel_to_wheel_vel(self.cmd_vel);
+        // let wheel_vels = self.robot_model.robot_vel_to_wheel_vel(self.cmd_vel);
```

Comment:
Should this be commented out?

Reply:
it can probably be deleted, but it lets us bypass controls I think.